src/System.Management.Automation/engine/ExperimentalFeature/GetExperimentalFeatureCommand.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// Implements Get-ExperimentalFeature cmdlet.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "ExperimentalFeature", HelpUri = "")]
+    [Cmdlet(VerbsCommon.Get, "ExperimentalFeature", HelpUri = "https://go.microsoft.com/fwlink/?linkid=2007351")]
```

Comment:
Please add a test for this by adding an entry at: https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/Help/assets/HelpURI/V3Cmdlets.csv

Reply:
Done. csv file updated.